embedchain/chunkers/base_chunker.py

```diff
@@ -25,9 +26,11 @@ def create_chunks(self, loader, src):
 
             for chunk in chunks:
                 chunk_id = hashlib.sha256((chunk + url).encode()).hexdigest()
-                ids.append(chunk_id)
-                documents.append(chunk)
-                metadatas.append(meta_data)
+                if (idMap.get(chunk_id) == None):
```

Comment:
when comparing to None, `is` is preferred.

Reply:
Done :+1: